### PR TITLE
feat(nft-reward): add endpoint

### PR DESCRIPTION
**Pull request by tk-o**
Resolves: #12 

The newly created endpoint `/api/create-erc721-metadata` creates markup for the NFT svg file, uploads this file to IPFS, and finally, returns a metadata object that covers all properties required by [the ERC-721 standard](https://eips.ethereum.org/EIPS/eip-721).

### Request example
image

### NTF svg file on IPFS example
image

### CODE REVIEW

File: modules/svg-bonanza/package.json [hunk 1]
```diff
@@ -8,11 +8,13 @@
     "start": "next start"
   },
   "dependencies": {
-    "next": "10.0.9",
+    "next": "10.0.4",
+    "nft.storage": "^0.3.10",
```
Comment (korzewski) [inline modules/svg-bonanza/package.json:12]: Maybe remove `^` and keep fix version "0.3.10"?

File: modules/svg-bonanza/src/pages/api/generate-nft-reward.ts [hunk 2]
```diff
@@ -0,0 +1,82 @@
+import * as path from 'path';
+import * as fs from 'fs/promises';
+import { NextApiRequest, NextApiResponse } from 'next';
+import { NFTStorage, Blob } from 'nft.storage';
+
+export default async function generateNftReward(
+  req: NextApiRequest,
+  res: NextApiResponse
+) {
+  const { reward, score, scoreOutOf, sessionId = 'skibidibiba' } = (req.body ||
```
Comment (korzewski) [inline modules/svg-bonanza/src/pages/api/generate-nft-reward.ts:10]: Remove sessionId fallback, this should be required param

File: modules/svg-bonanza/src/pages/api/generate-nft-reward.ts [hunk 3]
```diff
@@ -0,0 +1,82 @@
...
+  const svgMarkup = svgTemplate
+    .replace('{reward}', `${rewardFormatted}`)
+    .replace('{score}', score)
+    .replace('{scoreOutOf}', scoreOutOf)
+    .replace('{signature}', sessionId);
+
+  return svgMarkup;
+}
+
+function createMetadata(cid: string) {
```
Comment (korzewski) [inline modules/svg-bonanza/src/pages/api/generate-nft-reward.ts:63]: maybe we could rename it to `createERC721Metadata`? But it's up to you

File: modules/svg-bonanza/package.json (hunk 1 above)
Comment (tk-o) [reply modules/svg-bonanza/package.json:12]: It's fixed in the lockfile 😎

**Comment by tk-o** [comment]
All suggestion applied, @korzewski 🚀

**Review by deployAt — APPROVED** [review]

**Review by korzewski — APPROVED** [review]

**Comment by korzewski** [comment]
🚀

**Pull request merged**